.github/workflows/beta.yaml

```diff
@@ -39,19 +39,38 @@ jobs:
         with:
           useConfigFile: true
 
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
-
       - name: Login to the container registry
         uses: docker/login-action@v2
         with:
           registry: ${{ env.REGISTRY }}
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Prepare
+        id: prep
+        shell: bash
+        run: |
+          architecture=$(uname -m)
+          if [[ $architecture == "x86_64" ]]; then
+            echo "ARCH=amd64" >> $GITHUB_OUTPUT
+            echo "ARM=false" >> $GITHUB_OUTPUT
+          elif [[ $architecture == "aarch64" ]]; then
+            echo "ARCH=arm64" >> $GITHUB_OUTPUT
+            echo "ARM=true" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Generate docker image tags
+        id: metadata
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            name=ghcr.io/${{ github.repository_owner }}/porla-beta-${{ steps.prep.outputs.ARCH }}
```

Comment:
If we're not pushing the image, is it better to tag it without a domain and repository owner? Only `porla-beta-amd64` for example.